tests/states/upload.py

```diff
@@ -80,6 +80,9 @@ async def bg_upload(self, files: List[rx.UploadFile]):
         pass
 
 
+rx.fragment(FileUploadState.img_list)
```

Comment:
this would be more readable with a helper function, maybe defined in reflex.vars:

```
rx.vars.mark_used(FileUploadState.img_list)
```

This helper function should take arbitrary `*args` and just set `_var_is_used = True` on each of them.

Reply:
I agree, i had a bad feeling writing those lines